morerss/zhihu_stream.py

```diff
@@ -122,14 +182,21 @@ def post2rss(post, digest=False, pic=None):
     logger.warn('unknown type: %s', post['type'])
     return
 
-  if digest:
+  # Posts in Zhihu topics API response don't have the 'content' key by default
+  # Although they can include it by carrying verbose query params
+  # which are hard to maintain because Zhihu doesn't have public API documentation :(
+  if 'content' not in post or digest:
     content = post['excerpt']
   else:
     content = post['content']
 
   content = content.replace('<code ', '<pre><code ')
   content = content.replace('</code>', '</code></pre>')
 
+  # Post only contains images but no text
+  if not content:
```

Comment:
https://github.com/lilydjwg/morerssplz/pull/32/commits/6d01a6e1a23ba5f68628e03cc52a522b276f67c8 also addresses some posts that only contain images but no text in the Zhihu topics API.